styles.css

```diff
@@ -113,3 +110,229 @@ h2 {
   left: 65%;
   top: 46%;
 }
+
+#work {
+  display: flex;
+  flex-direction: column;
+  padding: 30% 6% 25% 6%;
+}
```

Comment:
- [ ] Good job wrapping everything in a container as required :clap:  , but the main container  of the projects should have a `display:grid` instead of `flex`. Kindly update that. 
Always see  requirements as customer needs you have to meet :+1: